test/cppsuite/tests/poc.cxx

```diff
@@ -1,26 +1,30 @@
 #include <iostream>
-#include <stdlib.h>
+#include <cstdlib>
+#include "test_harness/test_harness.h"
 
-extern "C" {
-    #include "wiredtiger.h"
-}
+class poc_test : public test_harness::test {
+    public:
+    int run() {
+        WT_CONNECTION *conn;
+        int ret = 0;
+        /* Setup basic test directory. */
+        const std::string default_dir = "WT_TEST";
```

Comment:
I think we'll want every test to use a unique directory (so we can run them in parallel).
That's future work though.